src/ai/backend/test/contexts/config.py

```diff
@@ -0,0 +1,90 @@
+from typing import override
+from uuid import UUID
+
+from ai.backend.test.contexts.context import BaseTestContext, ContextName
+from ai.backend.test.tester.config import (
+    BatchSessionConfig,
+    ClusterConfig,
+    EndpointConfig,
+    ImageConfig,
+    KeyPairConfig,
+    LoginCredentialConfig,
+    SessionConfig,
+    SessionTemplateConfig,
+    SSEConfig,
+)
+
+
+class KeypairConfigContext(BaseTestContext[KeyPairConfig]):
+    @classmethod
+    def name(cls) -> ContextName:
+        return ContextName.KEYPAIR
+
+
+class LoginCredentialConfigContext(BaseTestContext[LoginCredentialConfig]):
+    @classmethod
+    def name(cls) -> ContextName:
+        return ContextName.LOGIN_CREDENTIAL
+
+
+class EndpointConfigContext(BaseTestContext[EndpointConfig]):
+    @classmethod
+    def name(cls) -> ContextName:
+        return ContextName.ENDPOINT
+
```

Comment:
It seems like these values are not only used in config, but can also be set in `parametrize`. Would it be okay to move them all to the config package?

Reply:
In addition to parametrize, each value can also set values in the context from each template.

Reply:
That’s the part I’m currently thinking about.
I wanted to separate the context configured directly in the config from the context set in the template.
Is there no need to separate these?
Wouldn't it become difficult to trace where the context was configured?